website/content/community/maintainers.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Maintainers
+weight: 1
+---
+
+MetalLB was created by Dave Anderson.  We are thankful for his efforts to
+create MetalLB as a valuable contribution to the Kubernetes ecosystem.  Ongoing
+MetalLB development and maintenance is run by a team of current maintainers.
+
+MetalLB maintainers are participants in the project with the ability to approve
+changes to the MetalLB code base.  The current list of maintainers can be found
+in the [MetalLB OWNERS
+file](https://github.com/metallb/metallb/blob/main/OWNERS).
+
+## Maintainer Expectations
+
+An active maintainer should:
+
+* Actively participate in reviewing pull requests and incoming issues.  Note
+  that there are no hard rules on what is “active enough” and this is left up
+  to the judgement of the current group of maintainers.
+
+* Actively participate in discussions about design and the future of the
+  project.
+
+* Take responsibility for backports to appropriate braches for PRs they approve
```

Comment:
>Take responsibility for backports to appropriate branches for PRs they approve and merge

What do we mean by this? I feel like this may be too specific for a general explanation about the responsibilities of the maintainers.

Reply:
I mean that if a given maintainer is the one to merge a PR into `main`, it should be up to that maintainer to ensure that change is backported, if appropriate.  Does that make sense?

This assumes there is a defined release branch maintenance policy, clarifying what types of changes should be backported ... i'm not sure if that exists though?

Reply:
Makes sense to me. I don't think we have such a policy.